src/components/User/User.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+import { useDispatch, useStore } from "react-redux";
+
+import { logOut } from "app/actions";
+import useSendAnalytics from "app/send-analytics-hook";
+
+import "./_user.scss";
+
+const User = () => {
+  const sendAnalytics = useSendAnalytics();
+  const [isExpanded, setIsExpanded] = useState(false);
+  const dispatch = useDispatch();
+  const getState = useStore().getState;
+
+  useEffect(() => {
+    if (isExpanded) {
+      sendAnalytics({
+        category: "User",
+        action: "Opened user panel"
+      });
+    }
+  }, [isExpanded, sendAnalytics]);
+
+  return (
+    <div
+      className={classNames("user", {
+        "is-expanded": isExpanded
+      })}
+    >
+      <div className="user__header" onClick={() => setIsExpanded(!isExpanded)}>
+        <i className="p-icon--user"></i>
+        {/* @TODO Add user name here */}
```

Comment:
Is there a follow-up issue for this?

Reply:
https://github.com/canonical-web-and-design/jaas-dashboard/issues/365